code/datums/materials/basemats.dm

```diff
@@ -248,3 +248,143 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	. = ..()
 	if(istype(source, /obj/item))
 		qdel(source.GetComponent(/datum/component/fantasy))
+
+//I don't like sand. It's coarse, and rough, and irritating, and it gets everywhere.
+/datum/material/sand
+	name = "sand"
+	id = "sand"
+	desc = "You know, it's amazing just how structurally sound sand can be."
+	color = "#EDC9AF"
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/sandblock
+	value_per_unit = 0.0010
+	strength_modifier = 0.5
+	integrity_modifier = 0.1
+	armor_modifiers = list("melee" = 0.25, "bullet" = 0.25, "laser" = 1.25, "energy" = 0.25, "bomb" = 0.25, "bio" = 0.25, "rad" = 1.5, "fire" = 1.5, "acid" = 1.5)
+	beauty_modifier = 0.25
+	turf_sound_override = FOOTSTEP_SAND
+	texture_layer_icon_state = "sand"
+
+//And now for our lavaland dwelling friends, sand, but in stone form! Truly revolutionary.
+/datum/material/sandstone
+	name = "sandstone"
+	id = "sandstone"
+	desc = "Bialtaakid 'ant taerif ma hdha."
+	color = "#B77D31"
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/mineral/sandstone
+	value_per_unit = 0.0025
+	armor_modifiers = list("melee" = 0.5, "bullet" = 0.5, "laser" = 1.25, "energy" = 0.5, "bomb" = 0.5, "bio" = 0.25, "rad" = 1.5, "fire" = 1.5, "acid" = 1.5)
+	beauty_modifier = 0.3
+	turf_sound_override = FOOTSTEP_WOOD
+	texture_layer_icon_state = "brick"
+
+/datum/material/snow
+	name = "snow"
+	id = "snow"
+	desc = "There's no business like snow business."
+	color = "#FFFFFF"
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/mineral/snow
+	value_per_unit = 0.0025
+	armor_modifiers = list("melee" = 0.25, "bullet" = 0.25, "laser" = 0.25, "energy" = 0.25, "bomb" = 0.25, "bio" = 0.25, "rad" = 1.5, "fire" = 0.25, "acid" = 1.5)
+	beauty_modifier = 0.3
+	turf_sound_override = FOOTSTEP_SAND
+	texture_layer_icon_state = "sand"
+
+/datum/material/runedmetal
+	name = "runed metal"
+	id = "runed metal"
+	desc = "Mir'ntrath barhah Nar'sie."
+	color = "#3C3434"
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/runed_metal
+	value_per_unit = 0.75
+	armor_modifiers = list("melee" = 1.2, "bullet" = 1.2, "laser" = 1, "energy" = 1, "bomb" = 1.2, "bio" = 1.2, "rad" = 1.5, "fire" = 1.5, "acid" = 1.5)
+	beauty_modifier = -0.15
+	texture_layer_icon_state = "runed"
+
+/datum/material/bronze
+	name = "bronze"
+	id = "bronze"
+	desc = "Clock Cult? Never heard of it."
+	color = "#92661A"
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/tile/bronze
+	value_per_unit = 0.5
```

Comment:
worth way too much, you can pretty much have hundreds of bronze roundstart

Reply:
elasticity

Reply:
Not really, even with elasticity decreasing the cost bronze is very, very easy to produce early into the round (This makes one sheet of bronze worth 1000 credits). Still, considering it's supposed to be an aesthetic material, maybe a value around uranium's 0.05

Reply:
ah, didn't realise bronze was so readily available, will tweak this down